tests/drivers/gpio/gpio_enable_disable_interrupt/boards/native_posix.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	resources {
+		compatible = "test-gpio-enable-disable-interrupt";
+		in-gpios = <&gpio0 0 0>; /* Pin 0 */
```

Comment:
Nit - name `irq-gpios` to follow general Zephyr convention.

Reply:
Done